.github/workflows/build.yml

```diff
@@ -3,17 +3,44 @@ name: build
 
 on:
   push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
   pull_request:
+  schedule:
+    - cron: '0 10 * * *'  # everyday at 10am
   repository_dispatch:
+    # Respond to rebuild requests. See: https://github.com/cisagov/action-apb/
     types: [apb]
+  workflow_dispatch:
+    inputs:
+      remote-shell:
+        description: "Debug with remote shell"
+        required: true
+        default: false
+      image-tag:
+        description: "Tag to apply to pushed images"
+        required: true
+        default: dispatch
 
 env:
+  BUILDX_CACHE_DIR: ~/.cache/buildx
+<<<<<<< HEAD
   IMAGE_NAME: cisagov/postfix
+=======
+  IMAGE_NAME: cisagov/example
+>>>>>>> 9e993fbb4bbad507742670c91d98806190f37f0f
```

Comment:
Some merge markers were missed here.

Reply:
How did this even run?!  The lights were all green.  
Sometimes I amaze myself.